scripts/publish-staged

```diff
@@ -62,6 +62,8 @@ publish_s3() {
 			   "s3://${PUBLISH_S3_BUCKET}/${base_name}-${tag}.${suffix}"
 		dryval s3_cp "s3://${STAGE_S3_BUCKET}/${base_name}-${IMAGE_TAG_SHA}.${suffix}.md5" \
 			   "s3://${PUBLISH_S3_BUCKET}/${base_name}-${tag}.${suffix}.md5"
+        dryval s3_cp "s3://${STAGE_S3_BUCKET}/${base_name}-${IMAGE_TAG_SHA}.${suffix}.json" \
```

Comment:
Indentation looks wrong everywhere.

Reply:
Once you have chosen a file to inspect with a specific version, you can infer the version from the name, yes. However, if you dont know what version you want, or if you just want latest... this will allow you to check what the latest version is, check it against your version, and then decide if you want to pull 'latest' or not.